tools/generator/Tests/Unit-Tests/CodeGeneratorExpectedResults/XamarinAndroid/WriteInterfaceDefaultMethod.txt

```diff
@@ -0,0 +1,40 @@
+// Metadata.xml XPath interface reference: path="/api/package[@name='java.code']/interface[@name='IMyInterface']"
+[Register ("java/code/IMyInterface", "", "java.code.IMyInterfaceInvoker")]
+public partial interface IMyInterface : IJavaObject {
+	new static IntPtr class_ref = JNIEnv.FindClass ("java/code/IMyInterface");
```

Comment:
For our own sanity, why don't we restrict default-interface-method support to `CodeGenerationTarget.XAJavaInterop1` and `CodeGenerationTarget.JavaInterop1`, and *not* `CodeGenerationTarget.XamarinAndroid`.

That should allow us to remove some of these test files.

Reply:
Done, this combination now reports an error and exits.